src/main.rs

```diff
@@ -16,29 +16,51 @@ use tokio_core::reactor::Core;
 use name_gen::random_msg;
 
 fn main() {
-    let token = get_token(); 
-
+    let token = get_token();
     let epic = random_msg();
-    let tweet = DraftTweet::new(&epic);
-    
-    let mut core = Core::new().unwrap();
-    let handle = core.handle();
-    core.run(tweet.send(&token, &handle)).unwrap();
+
+    if let Err(token) = token {
+        println!("Couldn't post tweet ({}), reason: Failed to find an .env variable: {:?}", epic, token);
+    } else {
+        
+        let tweet = DraftTweet::new(&epic);
+        
+        let mut core = Core::new().unwrap();
+        let handle = core.handle();
+        let result = core.run(tweet.send(&token.unwrap(), &handle));
```

Comment:
You're still doing `.unwrap()`, if you want to use both the `Err` and `Ok` case, use a `match` instead.